homepage/neuroimaging.rst

```diff
@@ -26,21 +26,32 @@ The technology and physics behind an MRI scanner is quite astonishing. But I won
        :width: 160pt
        :align: center
 
-The volume data of a whole brain recorded at one single timepoint, as pictured on the left, is called a **volume** and consists of multiple voxels (pixels in 3D). Each **voxel** has a specific dimension, in this case it's an isotropic dimension of 1mm x 1mm x 1mm. Each voxel contains one value which stands for the average signal measured at the given location. A standard anatomical volume, with an isotropic voxel resolution of 1mm contains almost 17 million voxels, which are arranged in a **3D matrix** of 256 x 256 x 256 voxels. The following picture describes a vertical slice through such a brain volume and depicts in general the grid like structure of this 3D Matrix, consisting of multiple voxels.
+The brain occupies space, so when we collect data on how it fills space, we call that volume data, and all the volume data needed to create the complete, 3D image of the brain, recorded at one single timepoint and as pictured on the left, is called a **volume**.  The data is measured in **voxels**, which are like the pixels used to display images on your screen, only in 3D. Each voxel has a specific dimension, in this case it is 1mm x 1mm x 1mm: a cube, so it is the same dimension from all sides (isotropic). Each voxel contains one value which stands for the average signal measured at the given location.
+
+A standard anatomical volume, with an isotropic voxel resolution of 1mm contains almost 17 million voxels, which are arranged in a **3D matrix** of 256 x 256 x 256 voxels. The following picture shows a vertical slice -- one layer of the big, 3D grid looked at from the top -- through a brain volume, and the grid for the remaining two dimensions is superimposed on the picture of the brain.  The data is measured per box, and each of the boxes corresponds to a particular portion of the brain.
 
```

Comment:
I would rewrite the second paragraph a bit. What do you think of this (changes in italic):
"The following picture shows a _frontal_ slice -- one layer of the big, 3D grid looked at from the _front_ -- through a brain volume, and the grid for the remaining two dimensions is superimposed on the picture of the brain."

I'm not sure if I understand what you mean with a box in "The data is measured per box, and each of the boxes corresponds to a particular portion of the brain." Do you mean Voxels? In this case, I think we can drop the last sentence as voxels are explained just before, no?